tests/testthat/test-import_file.R

```diff
@@ -0,0 +1,9 @@
+test_that("file import runs successfully", {
+  vx <- import_file("dbtest/VACCINES.csv")
+  
+  expect_equal(class(vx), c("data.table", "data.frame"))
+  expect_equal(dim(vx), c(20, 12))
```

Comment:
Why 12 columns? I counted, and excluding the column X, there should be 11. The load_db function should only include the columns that are defined in the CDM_metadata file.